src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery1 = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
using numbers in naming is a bad practice